src/components/Editor/Tabs.js

```diff
@@ -85,6 +85,11 @@ const SourceTabs = React.createClass({
     if (!(prevProps === this.props)) {
       this.updateHiddenSourceTabs();
     }
+    const { selectedSource, sourceTabs, selectSource } = this.props;
+    if (!selectedSource && sourceTabs.size > 0) {
+      const source = sourceTabs.get(0);
+      selectSource(source.get("id"));
+    }
```

Comment:
I think you're right. I was unsure what the best would be to handle it. I removed it for now.